src/dashboard/Data/Browser/AddColumnDialog.react.js

```diff
@@ -206,6 +206,9 @@ export default class AddColumnDialog extends React.Component {
         confirmText='Add column'
         cancelText={'Never mind, don\u2019t.'}
         onCancel={this.props.onCancel}
+        continueText={'Create column & add another'}
```

Comment:
It should probably be named `Add column & continue...`, because:
- `Add column` makes it clearer that this executes the same as the button `Add column`.
- `& continue` is neutral and does not imply any action; while `add another` implies that something is added in addition
- `...` indicates that user input will be expected